Data/Services/CustomerService.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Data.Entities;
+
+namespace Data.Services
+{
+    public class CustomerService : ICustomerService
+    {
+        private List<Customer> _customerList = new List<Customer>();
+
+        public virtual void Create(Customer customer)
+        {
+            var checkingCustomerCpf = _customerList.FirstOrDefault(customerElement => customerElement.Cpf == customer.Cpf);
+            var checkingCustomerEmail = _customerList.FirstOrDefault(customerElement => customerElement.Email == customer.Email);
+            if (checkingCustomerCpf != null || checkingCustomerEmail != null) throw new ArgumentNullException("Customer already exists");
+
+            customer.Id = _customerList.Count + 1;
+            _customerList.Add(customer);
+        }
+
+        public virtual IEnumerable<Customer> GetAll()
+        {
+            return _customerList;
+        }
+
+        public virtual Customer GetById(long Id)
+        {
+            var customer = _customerList.FirstOrDefault(customer => customer.Id == Id);
+            if (customer != null) return customer;
+            throw new ArgumentException("Customer doesn't exist!");
```

Comment:
Não existe para o que? Devemos sempre especificar o problema com detalhes para que seja possível entender o erro
```suggestion
            throw new ArgumentException($"Customer does not exist for Id: {id}.");
```